edgemesh/pkg/server/dns.go

```diff
@@ -0,0 +1,401 @@
+package server
+
+import (
+	"bufio"
+	"encoding/binary"
+	"fmt"
+	"github.com/golang/glog"
+
+	//"github.com/golang/glog"
```

Comment:
please remove commented import and the blank line above

Reply:
ok，I will fix that